### PR TITLE
Passions Screen

### PR DESCRIPTION
**Pull request by cgntnr**
Passions Screen is added to the Profile Preferences section.

### CODE REVIEW

File: app/src/main/java/ch/epfl/sdp/blindly/Profile6.kt [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
 import android.os.Bundle
+import android.view.View
 
 class Profile6 : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.set_profile_6)
     }
+    fun start_profile_7(view: View) {
+        // Do something in response to button
```
Comment (koko-ng) [inline app/src/main/java/ch/epfl/sdp/blindly/Profile6.kt:14]: I think this comment could be deleted.

**Review by koko-ng — APPROVED** [review]
Alright for me.

File: app/src/main/java/ch/epfl/sdp/blindly/Profile6.kt (hunk 1 above)
Comment (cgntnr) [reply app/src/main/java/ch/epfl/sdp/blindly/Profile6.kt:14]: Yes you are right, I will delete this in the next PR.

**Review by Lollipoke — APPROVED** [review]
Seems good

**Pull request merged**